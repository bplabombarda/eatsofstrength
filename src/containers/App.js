import React from 'react';

import Plan from 'Components/Plan';

function App () {
  return (
    <section>
      <Plan/>
    </section>
  );
}

export default App;
