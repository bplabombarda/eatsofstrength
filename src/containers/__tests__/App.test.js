import React from 'react';

import App from '../App';

describe('App component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it('renders', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
