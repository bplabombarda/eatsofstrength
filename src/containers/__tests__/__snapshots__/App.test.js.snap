// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders 1`] = `
<section>
  <h1>
    Frigg off, Barb!
  </h1>
</section>
`;
